docs/content/editor-and-ide-support.md

```diff
@@ -16,6 +16,7 @@ evaluation, policy coverage, and more.
 | Atom | [https://github.com/open-policy-agent/opa/tree/master/misc/syntax/atom](https://github.com/open-policy-agent/opa/tree/master/misc/syntax/atom) |
 | Sublime Text | [https://github.com/open-policy-agent/opa/tree/master/misc/syntax/sublime](https://github.com/open-policy-agent/opa/tree/master/misc/syntax/sublime) |
 | TextMate | [https://github.com/open-policy-agent/opa/tree/master/misc/syntax/textmate](https://github.com/open-policy-agent/opa/tree/master/misc/syntax/textmate) |
+| Emacs | [https://github.com/psibi/rego-mode](https://github.com/psibi/rego-mode) |
```

Comment:
Can you move it up between `Atom` and `Sublime Text`? These were just in alphabetical order (not grouped any other way)